site/layouts/sbom/single.html

```diff
@@ -0,0 +1,45 @@
+{{ define "title" }}{{ .Site.Title }} &ndash; {{ .Title }}{{ end }}
+{{ define "main" }}
+{{ $addOn := index (where .Site.Data.addons "id" "==" $.Params.cascade.addon.id) 0 }}
+<section class="bolt-header">
+  <div class="wrapper py-20">
+    <h1 class="text--white">{{ .Title }}</h1>
+  </div>
+</section>
+<div class="wrapper py-20" data-kind="{{ .Kind }}">
+  <header class="breadcrumbs">
+    <a href="/docs/">Docs</a> &gt;
+    SBOM &gt;
+    <a href="/docs/sbom/{{ $addOn.id }}">{{ $addOn.name }}</a>
+  </header>
+  <br>
+  <p>This page contains a list of all the third-party libraries used by version <code>{{ $addOn.version }}</code> of the
```

Comment:
- The third-party should be removed IMO, it also lists our own libs.
- It would better to clarify what's included (compared to what's bundled, which just includes runtime).